### PR TITLE
feat(suite): haptic-feedback settings

### LINKED ISSUE #12314: Add option to enable/disable haptic feedback on T3T1 (UI)
With the arrival of T3T1, haptic feedback will be introduced as a device feature. Please implement an option to turn this feature on or off in Suite settings - Device - Customization section. Graphically, please use similar logic as with the PIN/Passphrase options.

Related to [this](https://github.com/trezor/trezor-suite/pull/12313) Connect PR

image

image

### PR DESCRIPTION
**Pull request by peter-sanderson**
## Description

- Adds a settings switch to allow haptic feedback (blocked by https://github.com/trezor/trezor-suite/pull/12313)
- Adds s setting for triggering the brightness flow on the device

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/12314 and https://github.com/trezor/trezor-suite/issues/12442

## Screenshots:

image
image

### CODE REVIEW

File: packages/suite-analytics/src/constants.ts [hunk 1]
```diff
@@ -67,6 +67,7 @@ export enum EventType {
     SettingsDeviceChangeLabel = 'settings/device/change-label',
     SettingsDeviceUpdateAutoLock = 'settings/device/update-auto-lock',
     SettingsDeviceChangeOrientation = 'settings/device/change-orientation',
+    SettingsDeviceChangeHapticFeedback = 'settings/device/change-haptic-feedback',
```
Comment (peter-sanderson) [inline packages/suite-analytics/src/constants.ts:70]: - [ ] add it to Notion docs for analytics

File: packages/suite/src/support/messages.ts [hunk 2]
```diff
@@ -2477,6 +2477,14 @@ export default defineMessages({
         defaultMessage: 'Change PIN',
         id: 'TR_DEVICE_SETTINGS_CHANGE_PIN_TITLE',
     },
+    TR_DEVICE_SETTINGS_HAPTIC_FEEDBACK_DESC: {
+        defaultMessage: "Enable haptic feedback for device interactions",
```
Comment (Hannsek) [inline packages/suite/src/support/messages.ts]: "Turn on" instead of "Enable"?

Comment (peter-sanderson) [reply packages/suite/src/support/messages.ts]: cc @Hermez-cz 

image


I accept the copy of the survivor ;)

Comment (Hermez-cz) [reply packages/suite/src/support/messages.ts]: Checking with Ben to make sure. Enable sounds better to me, turn on is more like turn on the vacuum cleaner.

Comment (Hermez-cz) [reply packages/suite/src/support/messages.ts]: Ben prefers `Turn on` by a tiny bit, so let's have `Turn on` in Suite as well.

Comment (peter-sanderson) [reply packages/suite/src/support/messages.ts]: done

File: packages/connect/src/types/api/index.ts [hunk 3]
```diff
@@ -66,12 +68,14 @@ import { requestWebUSBDevice } from './requestWebUSBDevice';
 import { resetDevice } from './resetDevice';
 import { rippleGetAddress } from './rippleGetAddress';
 import { rippleSignTransaction } from './rippleSignTransaction';
+import { setBrightness } from './setBrightness';
 import { setBusy } from './setBusy';
 import { setProxy } from './setProxy';
+import { showDeviceTutorial } from './showDeviceTutorial';
```
Comment (peter-sanderson) [inline packages/connect/src/types/api/index.ts:74]: Just sort alphabetically

**Review by mroz22 — APPROVED** [review]
looks good

**Pull request merged**